test/common/TestBed.cpp

```diff
@@ -276,7 +276,7 @@ namespace RcclUnitTesting
     InteractiveWait("Finishing PrepareData");
   }
 
-  void TestBed::ExecuteCollectives(std::vector<int> const &currentRanks, bool const useHipGraph)
+  void TestBed::ExecuteCollectives(std::vector<int> const &currentRanks, bool const useHipGraph, int const timeout)
```

Comment:
I don't think you should add this, because now you can have two different timeouts - the one that someone passes in explicitly, vs the one that you get from env var.

If someone wants to write a test that has a "custom" timeout, they can pop that env var for that one test, then reset it when they are done.